### PR TITLE
store/tikv: enlarge default committer-concurrency to avoid queuing

### PR DESCRIPTION
**Pull request by youjiali1995**
Signed-off-by: youjiali1995 <zlwgx1023@gmail.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/22516

Problem Summary:

The default committer-concurrency(16) is too small. It results in queuing in TiDB if a transaction has more than 16 requests.

### What is changed and how it works?

What's Changed:

Enlarge the default value to 128. Committer concurrency is introduced in v4.0 to reduce pressure(server is busy) on TiKV. However, I have met several times the performance is bad due to queuing in TiDB. I prefer enlarging the default value to making TiKV less pressure.

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

Side effects

### Release note 

- No release note

### CODE REVIEW

**Review by sticnarf — APPROVED** [review]

File: store/tikv/2pc.go [hunk 1]
```diff
@@ -1644,7 +1644,7 @@ func (batchExe *batchExecutor) process(batches []batchMutations) error {
 		}
 	}
 	close(exitCh)
-	metrics.TiKVTokenWaitDuration.Observe(batchExe.tokenWaitDuration.Seconds())
+	metrics.TiKVTokenWaitDuration.Observe(float64(batchExe.tokenWaitDuration.Nanoseconds()))
```
Comment (cfzjywxk) [inline store/tikv/2pc.go:1647]: The `tokenWaitDuration` is a duration type which has nano second count, but in https://github.com/pingcap/tidb/pull/17268 the observed value is changed to seconds(see the grafana picture in 17268). I'm a bit confused which is correct.

Comment (sticnarf) [reply store/tikv/2pc.go:1647]: The bucket is defined to be `Buckets:   prometheus.ExponentialBuckets(1, 2, 34), // 1ns ~ 8s`. The base is 1, so it should be a nanosecond.

Comment (youjiali1995) [reply store/tikv/2pc.go:1647]: It's expected to be observed in nanoseconds. #17268 doesn't fix this. The root cause is the mismatch between the metric and the grafana panel.
```go
	TiKVTokenWaitDuration = prometheus.NewHistogram(
		prometheus.HistogramOpts{
			Namespace: namespace,
			Subsystem: subsystem,
			Name:      "batch_executor_token_wait_duration",
			Buckets:   prometheus.ExponentialBuckets(1, 2, 34), // 1ns ~ 8s
			Help:      "tidb txn token wait duration to process batches",
		})
```

**Comment by youjiali1995** [comment]
@cfzjywxk PTAL

**Review by cfzjywxk — APPROVED** [review]
LGTM

**Pull request merged**